RecoTracker/MkFit/test/reco_harvest_cfg.py

```diff
@@ -0,0 +1,107 @@
+# Auto generated configuration file
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: step4 --conditions auto:phase1_2021_realistic -s HARVESTING:@trackingOnlyValidation+@trackingOnlyDQM --scenario pp --filetype DQM --geometry DB:Extended --era Run3 --mc -n 100 --no_exec --filein file:step3_inDQM.root --fileout file:step4.root
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+
+process = cms.Process('HARVESTING',Run3)
```

Comment:
is there a way to get this directly from cmsDriver?

Reply:
The harvesting configuration file is directly from 11634.1, so yes, it can be obtained directly from `cmsDriver`.

In principle `harvestTrackValidationPlots.py` script would also do the job.

Having an explicit file was sort of a personal preference for a CMSSW-version-wise stable setup. (and what we have in 10_4_0)